### ISSUE TITLE
Add JSDoc to `CloseButton` component.

### ISSUE THREAD

**Issue by hirotomoyamada** [yamada-ui/yamada-ui#1221]
### Description

JSDoc is currently missing from the `CloseButton` component.

### Problem Statement/Justification

Since JSDoc exists for other components, we should also add `CloseButton` component.

### Proposed Solution or API

```diff
+ /**
+ * `CloseButton` is a component used primarily to trigger the close functionality of a component.
+ *
+ * @see Docs https://yamada-ui.com/components/other/close-button
+ */
export const CloseButton = forwardRef<CloseButtonProps, "button">(
  (props, ref) => {
    const [styles, mergedProps] = useComponentStyle("CloseButton", props)
    const {
      className,
      children,
      isDisabled,
      isRounded,
      __css,
      disableRipple,
      ...rest
    } = omitThemeProps(mergedProps)
    const { onPointerDown, ...rippleProps } = useRipple({
      ...rest,
      isDisabled: disableRipple || isDisabled,
    })

    const css: CSSUIObject = {
      position: "relative",
      overflow: "hidden",
      outline: 0,
      display: "flex",
      justifyContent: "center",
      alignItems: "center",
      flexShrink: 0,
      ...styles,
      ...__css,
      ...(isRounded ? { borderRadius: "fallback(full, 9999px)" } : {}),
    }

    return (
      <ui.button
        ref={ref}
        type="button"
        aria-label="Close"
        className={cx("ui-close-button", className)}
        disabled={isDisabled}
        __css={css}
        {...rest}
        onPointerDown={onPointerDown}
      >
        {children || <CloseIcon width="1em" height="1em" />}

        <Ripple isDisabled={disableRipple || isDisabled} {...rippleProps} />
      </ui.button>
    )
  },
)
```

### Are you willing to participate in implementing this feature and create a pull request include the implementation?

None

**Comment by Tay3052** [comment]
Can I join this?

**Comment by hirotomoyamada** [comment]
@Tay3052 

Hi, thanks for the comment!

Yes, pls.

**Comment by hirotomoyamada** [comment]
Fixed #1232
